heron/common/src/cpp/network/server.h

```diff
@@ -205,18 +205,6 @@ class Server : public BaseServer {
   // Called when the connection is closed
   virtual void HandleConnectionClose_Base(BaseConnection* connection, NetworkErrorCode _status);
 
-  MemPool<google::protobuf::Message> _heron_message_pool;
```

Comment:
the method `InstallRequestHandler` and `InstallMessageHandler` on line 150 and 160 are also using the `new` and `delete` operator manage message memory, should we also update them？